editor/components/page-attributes/template.js

```diff
@@ -0,0 +1,67 @@
+/**
+ * External dependencies
+ */
+import { connect } from 'react-redux';
+import { isEmpty, map } from 'lodash';
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { withContext, withInstanceId } from '@wordpress/components';
+import { compose } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import './style.scss';
+import { getEditedPostAttribute } from '../../store/selectors';
+import { editPost } from '../../store/actions';
+
+export function PageTemplate( { availableTemplates, selectedTemplate, instanceId, onUpdate } ) {
+	if ( isEmpty( availableTemplates ) ) {
+		return null;
+	}
+	const selectId = `template-selector-${ instanceId }`;
+	const onEventUpdate = ( event ) => onUpdate( event.target.value );
+	return (
+		<div className="editor-page-attributes__template">
+			<label htmlFor={ selectId }>{ __( 'Template:' ) }</label>
+			<select
+				id={ selectId }
+				value={ selectedTemplate }
+				onBlur={ onEventUpdate }
+				onChange={ onEventUpdate }
+			>
+				{ map( { '': __( 'Default template' ), ...availableTemplates }, ( templateName, templateSlug ) => (
```

Comment:
Nit: space missing after `''`, before `:`.

Reply:
If we add the space e.g make things like ''' : __( 'Default template' )' we get a linting error because of the extra space.

Reply:
Skip that, I thought it is `if` statement :)